lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -116,10 +116,29 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     if (fci) v5andv6metrics.push(fci);
     if (fmp) v5andv6metrics.push(fmp);
 
+    /** @type {Record<string, string>} */
+    const acronymMapping = {
```

Comment:
this seems like it could be useful a lot of place haha

Reply:
I would have added it to the audit meta but that had TS implications I didn't want to address.

Reply:
yup i think we'll eventually have to publish a module with it

https://github.com/paulirish/lh-scorecalc/blob/372ccbde31b19b896de350b5c32777c18fa7bd25/script/metrics.js#L2-L10